### ISSUE TITLE
Coup success rate

### ISSUE THREAD

**Issue by Edaka** [LoneGazebo/Community-Patch-DLL#937]
Originally posted by glaivemaster: "I've been playing on the recent betas, and I believe that one of them mentioned fixing the percentages on City State coups. However, since this change it seems to me that coups are all experiencing a 0% success rate, or at the very least an incredibly high failure rate to what is advertised. I'm happy to accept that 85% isn't 100%, but I don't think I've had any of my coups be successful since this change, and nor have the AI. Just constantly failed coups everywhere. Is anyone else experiencing this problem, or is there a further hidden mechanic I'm not noticing for why all my coups fail even on an 85% chance?"
 
tl;dr: coups with a 85% success chance fail way too frequently.
